pareto/strategic_water_management/strategic_produced_water_optimization.py

```diff
@@ -934,114 +1086,146 @@ def create_model(df_sets, df_parameters, default={}):
         model.s_T,
         default=0,
         initialize=model.df_parameters["CompletionsDemand"],
-        doc="Completions water demand [bbl/week]",
+        units=model.user_units["volume"] / model.user_units["time"],
```

Comment:
In regards to your scaling comment. It does seem like unit conversion significantly helps with stability of solutions. I've removed the rounding to 0 prior to solving the water quality model and it solves with no infeasibility !! I do believe it is still beneficial to keep scaling. There are still some large coefficients even with the units implementation (example PipelineExpansion Capex coefficient rage is [0, 106,640]). While not as large as before, it could still be beneficial. 

In regards to your suggestion to rewrite parameters to be in model units. I like this suggestions. Implementing this would mean our constraints are much more familiar to before. I wonder if it may also decrease model build time as conversion is only needed once.